packages/components/molecules/f-navigation-links/src/components/NavigationLinks.vue

```diff
@@ -1,32 +1,78 @@
 <template>
-    <div
+    <ul
+        v-if="links.length"
         :class="$style['c-navigationLinks']"
         data-test-id="navigationLinks">
-        I am a NavigationLinks Component (GB)
-    </div>
+        <li
+            v-for="({ id, url, name, selected }, i) in links"
+            :key="i"
+            :class="$style['c-navigationLink-item']">
+            <span
+                v-if="selected"
+                :data-test-id="id"
+                tabindex="0"
+                :class="[$style['c-navigationLink-link'], $style['c-navigationLink--active']]">
```

Comment:
So the base class is `c-navigationLinks-link` here and the modifier is indicated by the double dash which would be the active state `c-navigationLinks-link--active`. (Just a naming thing but can be easier to explain sometimes in person, so give me a shout if this doesn't make any sense!)

```suggestion
                :class="[$style['c-navigationLinks-link'], $style['c-navigationLinks-link--active']]">
```